tools/CAT/src/components/AddButton/index.jsx

```diff
@@ -0,0 +1,8 @@
+export default function AddButton({className = ''}){
+	return(
+		<div className={`flex bg-activeItems hover:bg-hoverActiveItems w-fit mx-auto text-white rounded-md items-center justify-center cursor-pointer px-3 ${className}`}>
+			<i className="fa-solid fa-plus text-3xl"></i>
+			<span className="text-lg ml-3">Додати</span>
+		</div>
+	)
+}
```

Comment:
Let`s Rename this component to TSX and u need to use some interface for this kind of props

for ex 

```
interface BaseUIComponentProps {
   className?: string;
}
```

and this Interface u can use in different UI components